src/plugins/intel_gpu/src/graph/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -35,9 +35,17 @@ std::vector<layout> experimental_detectron_roi_feature_extractor_inst::calc_outp
         experimental_detectron_roi_feature_extractor_node const& /*node*/, const kernel_impl_params& impl_param) {
     layout rois_layout = impl_param.get_input_layout(0);
     layout data_layout = impl_param.get_input_layout(1);
+    auto desc = impl_param.typed_desc<experimental_detectron_roi_feature_extractor>();
+    if (rois_layout.is_dynamic() || data_layout.is_dynamic()) {
```

Comment:
No need to gave special path for dynamic shapes here. The code below will work for both dynamic and static shapes